src/bp/admin/ui/package.json

```diff
@@ -69,6 +69,9 @@
     "npm-run-all": "^4.1.5",
     "react-app-rewired": "^2.1.0"
   },
+  "resolutions": {
+    "**/is-svg": "^4.2.2"
```

Comment:
Why not be more specific with packages you are targeting?
e.g.
```
"some-package/is-svg": "^4.2.2"
or
"some-package/**/is-svg": "^4.2.2"
```

Reply:
My thinking is I want to make sure is-svg < 4.2.2 is never used no matter the dependency.

Reply:
Ah ok makes sense 👌

Reply:
> My thinking is I want to make sure is-svg < 4.2.2 is never used no matter the dependency.

Maybe you should use >=4.2.2 then? This way, any dependencies can install v5 when it's released!